/**:
  ros__parameters:
    max_task_length: 5
    compressor:
      type: jpeg
      quality: 90
    rectifier:
      do_rectify: true
      alpha: 0.0
